README.md

```diff
@@ -110,9 +110,64 @@ your [development environment](https://hyperledger-fabric.readthedocs.io/en/rele
 
 Moreover, we assume that you are familiar with the Intel SGX SDK.
 
+### **Initial Setup**
+
+#### 1. Intel SGX SDK and SSL
```

Comment:
With option 2 you wouldn't need SDK or SGXSSL installation and PSW only (on host) if you want to run in HW mode.    So ideally we would talk here only about SGX & PSW and mention SGX_MODE but would outsource SGX SDK and SGXSSL to subsection in option 2?